proxy/src/auth/backend/jwt.rs

```diff
@@ -357,6 +358,20 @@ impl JwkCache {
     }
 }
 
+impl Default for JwkCache {
+    fn default() -> Self {
+        let client = Client::builder()
+            .user_agent(JWKS_USER_AGENT)
+            .redirect(redirect::Policy::none())
+            .build()
+            .unwrap();
```

Comment:
should be safe to unwrap this, since we're using &str and standard redirect::Policy

Reply:
nit: typically you can use `.expect("reason that you dont expect a panic")` instead of unwrap()

Reply:
yes, in fact, we would prefer that in at least proxy codebase.